.php_cs.dist

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Rules we follow are from PSR-2 as well as the rectified PSR-2 guide.
```

Comment:
Is there a blueprint where you got this whole fle from?

Reply:
https://gist.github.com/codfish/c77d348820c1c6b4ebe4a66dc2291c74#file-php_cs-dist